book/accessibility.md

```diff
@@ -698,6 +638,133 @@ sheets, and instead rely on extensions.
 
 :::
 
+Customizing dark mode
```

Comment:
Question: I organized the chapter previously around this order:

Section 1. Some a11y challenges, and how the browser addreseses them
Section 2. APIs for developers to customize.

I think this changes that right? Do you think it'd be better to restructure?

Reply:
You do make a good point though that this change reduces redundancy in the chapter...